src/main/java/com/arangodb/velocypack/VPackSlice.java

```diff
@@ -761,10 +885,18 @@ private int compareString(final String s) {
 		}
 	}
 
-	protected byte[] getRawVPack() {
+	protected byte[] getVPack() {

```

Comment:
This is never used. Can we remove it? Or you left it here for a specific reason?

Reply:
It would be a useful utility function when you need to get only the relevant part of the buffer.

I now noticed there is also the getBuffer method, so I got rid of the getRawVPack one.